_data/simple-icons.json

```diff
@@ -355,6 +355,11 @@
             "hex": "2A73CC",
             "source": "https://about.coursera.org/press"
         },
+        {
+            "title": "Creative Cloud",
+            "hex": "DA1F26",
+            "source": "https://www.adobe.com/legal/permissions/icons-web-logos.html"
```

Comment:
Note that the [Creative Cloud-specific guidelines](https://www.adobe.io/apis/creativecloud/creativesdk/docs/websdk/adobe-creative-sdk-for-web_master/branding-guidelines.html) contains an "on-tile" and an "off-tile" variant (with the off-tile version being the inverse of the version in this PR). However the [official website](https://www.adobe.com/dk/creativecloud.html) uses the variant in this PR, and I therefore vote that we stick with that.